app/controllers/admin/statistics_announcement_publications_controller.rb

```diff
@@ -0,0 +1,61 @@
+class Admin::StatisticsAnnouncementPublicationsController < Admin::BaseController
+  before_action :find_statistics_announcement
+  layout "design_system"
+
+  def index
+    if params[:search].present?
+      filter
+    end
+  end
+
+  def connect
+    @statistics_announcement.assign_attributes(publication_params)
+
+    if @statistics_announcement.save
+      redirect_to [:admin, @statistics_announcement], notice: "Announcement updated successfully"
+    else
+      filter
+      render :index
+    end
+  end
+
+private
+
+  def filter
+    @filter ||= Admin::EditionFilter.new(get_editions, current_user, edition_filter_options)
+  end
+
+  def get_editions
+    Edition.statistical_publications.with_title_containing(params[:search])
+  end
+
+  def params_filters
+    params.slice(:type, :state, :organisation, :author, :page, :title, :world_location, :from_date, :to_date, :only_broken_links)
+          .permit!
+          .to_h
```

Comment:
So i think we can simplify things here a bit. If we update the name of the search field to title, we can pass that into the edition filter and it will do the work to filter on title for us

```
def params_filters
    params.slice(:title)
          .permit!
          .to_h
end
````
then we can pass the publication type and subtype in as filters like this
```
def edition_filter_options
    filter_options = params_filters_with_default_state
                       .symbolize_keys
                       .merge(
                         include_unpublishing: true,
                         include_link_check_reports: true,
                         include_last_author: true,
                         type: "publication",
                         subtypes: @statistics_announcement.publication_type_id,
                       )

    filter_options.merge(per_page: Admin::EditionFilter::GOVUK_DESIGN_SYSTEM_PER_PAGE)
  end

```